sdk/freertos_app_cpu0/src/FreeRTOSConfig.h

```diff
@@ -0,0 +1,245 @@
+/* README!!!
+ *
+ * This configuration is copied from the FreeRTOS v10.1.1 based example project for the Zynq 7000 platform (for
+ * compatibility). However, you can add to it.
+ *
+ * Another example configuration file is available at
+ * "sdk/FreeRTOS-Kernel/examples/sample_configuration/FreeRTOSConfig.h" Because that example is in the submodule, it may
+ * update from time-to-time when the submodule is updated.
+ */
+
+/*
+ * Copyright (C) 2018 Amazon.com, Inc. or its affiliates.  All Rights Reserved.

```

Comment:
Lol "Amazon.com"

Reply:
Should we/can we remove these? They're kinda ugly but it feels illegal to remove a copyright. I'll leave unresolved.

Reply:
Let's leave it